raiden/blockchain_events_handler.py

```diff
@@ -283,10 +283,15 @@ def handle_secret_revealed(raiden, event):
 
 
 def on_blockchain_event(raiden, event):
-    log.debug('BLOCKCHAIN EVENT', node=pex(raiden.address), chain_event=event)
+    data = event.event_data
+    log.debug(
+        'BLOCKCHAIN EVENT',
+        node=pex(raiden.address),
+        contract=data['address'],
```

Comment:
Should this be "pexed" as well?

Reply:
Not sure, I like to have the full address for grepping, especially when it's only a debug level message. But no hard feelings on that.

Reply:
If it's not pexed it won't be readable. You need to pex it. It's binary otherwise.